utils/resb/src/bundle.rs

```diff
@@ -41,6 +44,9 @@ impl<'a> ResourceBundle<'a> {
     }
 
     /// Gets the name of the resource bundle.
+    ///
+    /// This name is used as the "key" of the root resource in a text format
+    /// bundle, but is not used in building binary resource bundles.
     pub fn name(&self) -> &str {
         &self.name
     }
```

Comment:
`name()` and `root()` I've left as getters at this time. Corresponding setters should be added, and those setters will need to filter out values which violate the de facto specification.